docs/Networking.asciidoc

```diff
@@ -82,299 +70,52 @@ firewall is not covered in all necessary details in this documentation.
-`firewall-cmd --zone=trusted --list-all`. It should show the configured interfaces
-and `masquerade: yes`. The firewall is configured this way by the previously
-mentioned salt states.
-
-You can of course also use `firewall-cmd` manually and use different zones. For
-instance, you can assign the bridge interface to the internal zone and the interface
-with access to the network to the external zone:
-
-[source,sh]
-----
-firewall-cmd --zone=external --add-interface=eth0
-firewall-cmd --zone=internal --add-interface=br1
-----
-
-To enable the virtual machines used by openQA to fully access the external
-network masquerading needs to be enabled on all involved zones:
-
-[source,sh]
-----
-firewall-cmd --zone=external --add-masquerade
-firewall-cmd --zone=internal --add-masquerade
-----
-
-IP forwarding is enabled automatically if masquerading is enabled:
-
-[source,sh]
-----
-grep 1 /proc/sys/net/ipv4/ip_forward
-1
-----
-
-In case the interface is in a trusted network it is possible to accept
-connections by default by changing the zone target:
-
-[source,sh]
-----
-firewall-cmd --zone=external --set-target=ACCEPT
-----
-
-If you are happy with the changes make them persistent:
-
-[source,sh]
-----
-firewall-cmd --runtime-to-permanent
-----
-
-If you do not currently have the firewalld service running, you can instead
-use the `firewall-offline-cmd` command for the configuration. In this case
-start the firewall and enable the service to run on system startup:
-
-[source,sh]
-----
-systemctl enable --now firewalld
-----
-
-Also, the `firewall-config` GUI tool for firewalld can be used for configuration.
-
-==== For older versions of openSUSE/SLE: Configure NAT with SuSEfirewall2
-
-The IP 10.0.2.2 can be also served as a gateway to access the outside network.
-For this, NAT between _br1_ and _eth0_ must be configured with SuSEfirewall2
-or iptables:
-
-[source,sh]
-----
-# /etc/sysconfig/SuSEfirewall2
-FW_DEV_INT="br1"
-FW_ROUTE="yes"
-FW_MASQUERADE="yes"
-----
-
-Start SuSEfirewall2 and enable the service to start on system startup:
-
-[source,sh]
-----
-systemctl enable --now SuSEfirewall2
-----
-
-
-==== Configure openQA workers
-Allow worker intstances to run multi-machine jobs:
-
-[source,sh]
-----
-# /etc/openqa/workers.ini
-[global]
-WORKER_CLASS = qemu_x86_64,tap
-----
-
-NOTE: The number of tap devices should correspond to the number of the running
-worker instances. For example, if you have set up 3 tap devices, the same
-number of worker instances should be configured.
-
-Enable worker instances to be started on system boot:
-
-[source,sh]
-----
-systemctl enable openqa-worker@1
-systemctl enable openqa-worker@2
-systemctl enable openqa-worker@3
-----
-
-==== Grant CAP_NET_ADMIN Capabilities to QEMU
-In order to let QEMU create TAP devices on demand it is required to set
-CAP_NET_ADMIN capability on QEMU binary file:
-
-[source,sh]
-----
-zypper -n in libcap-progs
-setcap CAP_NET_ADMIN=ep /usr/bin/qemu-system-x86_64
-----
```

Comment:
It doesn't look like this is done by the new script at all so we should rather preserve this section (as we'd otherwise lose this information completely). Or is it really completely outdated or never wanted to consider this kind of setup?

Reply:
apparently outdated and not needed anymore, see https://github.com/os-autoinst/os-autoinst/pull/2361